mirage/factories/article.js

```diff
@@ -1,11 +1,30 @@
-import { Factory } from 'ember-cli-mirage';
+import { Factory, association, faker } from 'ember-cli-mirage';
 
 export default Factory.extend({
-  description: 'description',
+  author: association(),
 
-  title: 'title',
+  title() {
+    return faker.lorem.words();
+  },
 
-  slug(i) {
-    return 'slug-' + i;
+  tagList: ['dragons', 'training'],
```

Comment:
Why not generate these? Also so it isn't a fixed number of tags and random tags?